From 7e385c5b641e86760d15113d542424fb63335b01 Mon Sep 17 00:00:00 2001
From: Nehal J Wani <nehaljw.kkd1@gmail.com>
Date: Sun, 3 May 2020 12:35:30 -0400
Subject: [PATCH 3/3] Relax tolerance in test_volume_from_surface

---
 test/test_volume_from_surface.py | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/test_volume_from_surface.py b/test/test_volume_from_surface.py
index b267718..1f11b4b 100644
--- a/test/test_volume_from_surface.py
+++ b/test/test_volume_from_surface.py
@@ -18,6 +18,8 @@ def test_volume_from_surface():
     tol = 1.0e-3
     assert abs(max(mesh.points[:, 0]) - 0.357612477657) < tol
     assert abs(min(mesh.points[:, 0]) + 0.358747130015) < tol
+
+    tol = 1.0e-1
     assert abs(max(mesh.points[:, 1]) - 0.496137874959) < tol
     assert abs(min(mesh.points[:, 1]) + 0.495301051456) < tol
     assert abs(max(mesh.points[:, 2]) - 0.298780230629) < tol
-- 
2.21.1

